src/js/components/action.jsx

```diff
@@ -17,6 +18,23 @@ export class ActionComponent extends Component {
         };
     }
 
+    onPostbackClick = () => {
+        this.setState({
+            state: 'processing'
+        });
+
+        postPostback(this.props._id).then(() => {
+            this.setState({
+                state: 'paid'
```

Comment:
I was getting ready in case we were saving the state, those are the one in the `enum`. *shrug Sorry